### PR TITLE
FINERACT-1853: Fix general accounting reports reconciliation

### PR DESCRIPTION
**Pull request by josehernandezfintecheandomx**
## Description

Trial Balance Summary and Transaction Summary reports data are not reconciling with one and another, the Expectation is both the report values should match.

[FINERACT-1853](

https://issues.apache.org/jira/browse/FINERACT-1853)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Comment by bharathcgowda** [comment]
@adamsaghy  could you please help in the review of this PR?

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/0097_update_accounting_summary_table_reports.xml [hunk 1]
```diff
@@ -0,0 +1,915 @@
...
+                value
+              from
+                m_payment_type
+              where
+                id = a.payment_type_id
+            ) as PaymentType_Name,
+            a.chargetype as chargetype,
+            a.reversal_indicator AS Reversed,
+            a.Allocation_Type AS Allocation_Type,
+            '' AS Chargeoff_ReasonCode,
```
Comment (adamsaghy) [inline fineract-provider/src/main/resources/db/changelog/tenant/parts/0097_update_accounting_summary_table_reports.xml:51]: empty `Chargeoff_ReasonCode`?

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/0097_update_accounting_summary_table_reports.xml [hunk 2]
```diff
@@ -0,0 +1,915 @@
...
+            AND a.reversal_indicator = true then sum(a.amount) * -1 end AS Transaction_Amount
+            FROM
+            (
+              SELECT
+                '${endDate}' AS transactiondate,
+                t.id,
+                l.name AS product,
+                t.transaction_type_enum AS transaction_type,
+                d.payment_type_id,
+                '' as chargetype,
```
Comment (adamsaghy) [inline fineract-provider/src/main/resources/db/changelog/tenant/parts/0097_update_accounting_summary_table_reports.xml:65]: empty `chargetype`?

**Comment by adamsaghy** [comment]
@josehernandezfintecheandomx is this tested on mysql and postgres as well?

**Pull request merged**